src/components/registration-lite.js

```diff
@@ -306,7 +307,7 @@ const RegistrationLite = (
                                     />
                                 )}
 
-                                {profileData && step !== 3 && allowedTicketTypes.length > 0 && (
+                                {profileData && step !== 3 && (
```

Comment:
please use the defined constant for the steps not magic numbers ( 3 )

Reply:
also what is the rationale of removing allowedTicketTypes.length > 0  condition ?

Reply:
Replaced numbers for step with constants. 

allowedTicketTypes validation is removed because when the summit data is updated, it triggers the updated for ticket types and taxes, and while this is runnning, the value of allowedTicketTypes is [], so it causes the widget to hide all the form.